### DISCUSSION TITLE
Fonts issue ?

### DISCUSSION THREAD

**Discussion by quimperois** [Stirling-Tools/Stirling-PDF#1038] [Q&A]
Hi All ; someone has experimented the same issue : watermarked pdf with numbers result in invalid caracters but only numbers not letters ?? Both docker and web ; Sorry for my english , i'm french....

**Comment by Frooodle** [comment]
Do you have any example file and inputs we can test with to reproduce the issue

**Comment by quimperois** [comment]
[test.pdf](https://github.com/Stirling-Tools/Stirling-PDF/files/14896762/test.pdf)
[test_watermarked.pdf](https://github.com/Stirling-Tools/Stirling-PDF/files/14896764/test_watermarked.pdf)

watermarked text is "demo 2024"
Thanks for all your work Frooodle;-)
